src/controllers/openid4vc/holder/holder.service.ts

```
@@ -0,0 +1,254 @@
+  }
+
+  public async getMdocCredentials(agent: Agent) {
+    return await agent.mdoc.getAll()
+  }
+
+  // public async resolveCredentialOffer(agent: Agent, body: ResolveCredentialOfferBody) {
+  //   return await agent.modules.openId4VcHolderModule.resolveCredentialOffer(body.credentialOfferUri)
+  // }
+
+  public async requestAuthorizationForCredential(agent: Agent, body: AuthorizeRequestCredentialOffer) {
+    console.log('Requesting authorization for credential offer:', body)
```

ESLint

[error] Unexpected console statement.

(no-console)